extensions/mongo.md

```diff
@@ -16,11 +16,15 @@ Note that there is always a balance between query optimization and update speed.
 
```

Comment:
Seeing the introduction of this page, I think it may deserve some love, actually...
Granted, this should be sufficient for now. Especially with the intended documentation adjustments in the future.
So, this is more a marker comment to share my thoughts then something you should process.